ansible_collections/arista/cvp/molecule/cv_device_v3/test_cv_device_v3.yaml

```diff
@@ -0,0 +1,308 @@
+---
+- name: Test cv_device_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    CVP_DEVICES_MOVE_DEVICE:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
+
+    CVP_DEVICES_PARENT_CONTAINER_NONE:
+      - fqdn: leaf1
+        parentContainerName: None
+
+    CVP_DEVICES_PARENT_CONTAINER_SAME:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
+    CVP_DEVICES_STATE_ABSENT:
+      - fqdn: leaf2
+        parentContainerName: ""
+
+    CVP_DEVICES_FACT0RY_RESET:
+      - fqdn: leaf2
+        parentContainerName: ""
+
+    CVP_DEVICES_PROVISIONING_RESET:
+      - fqdn: leaf3
+        parentContainerName: ""
+
+    CVP_DEVICES_BUNDLE_ATTACHED:
+      - serialNumber: JPE504a004ea054
+        parentContainerName: L2_Leaf
```

Comment:
Did we create this container or is it something that is assumed to exist on ATD by default?

Reply:
fixed